### PR TITLE
Copy to clipboard for access commands

### PR DESCRIPTION
**Pull request by davet1985**
* using a read-only input with an inline button and tooltip
* extracted as a partial so it can be re-used elsewhere
* currently done for kubernetes, quay and ECR

Screen Shot 2019-10-15 at 14 14 03

Closes https://github.com/appvia/appvia-hub/issues/237

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,5 +50,6 @@
     "pre-commit": "^1.2.2",
     "prettier": "^1.18.2",
     "webpack-dev-server": "^3.7.2"
-  }
+  },
+  "version": "1.19.1"
```
[the remark this answers is not in the document]
Comment (danielwhatmuff) [reply package.json]:
```
$ npm version
{ 'appvia-hub': '1.19.1',
  npm: '6.11.3',
  ares: '1.14.0',
  cldr: '33.1',
  http_parser: '2.8.0',
  icu: '62.1',
  modules: '64',
  napi: '3',
  nghttp2: '1.34.0',
  node: '10.13.0',
  openssl: '1.1.0i',
  tz: '2018e',
  unicode: '11.0',
  uv: '1.23.2',
  v8: '6.8.275.32-node.36',
  zlib: '1.2.11' }
```

[the remark this answers is not in the document]
Comment (davet1985) [reply package.json]: This is a mistake 🙈

**Comment by danielwhatmuff** [comment]
link #237

**Comment by davet1985** [comment]
> link #237

Already did this in the description

File: app/webpack/packs/application.js [hunk 2]
```diff
@@ -66,4 +68,15 @@ document.addEventListener('turbolinks:load', () => {
   // Make bootstrap-select work with Turbolinks
   $(window).trigger('load.bs.select.data-api');
 });
+
+$(document).ready(function() {
```
Comment (jits) [inline app/webpack/packs/application.js]: @davet1985 – unfortunately this doesn't play well with Turbolinks… to reproduce, load up any page apart from the Access page, then navigate to the Access page via the link in the sidebar, now try to use the clipboard button (it won't work). To fix this, you need to add the code below to the `turbolinks:load` event handler above.

Comment (davet1985) [reply app/webpack/packs/application.js]: Ok, makes sense will fix this.

**Pull request merged**